LKExp-Firmware-Disco-Pelion_via_ethernet/update_default_resources.c

```diff
@@ -0,0 +1,91 @@
+
+//----------------------------------------------------------------------------
+//   The confidential and proprietary information contained in this file may
+//   only be used by a person authorised under and to the extent permitted
+//   by a subsisting licensing agreement from ARM Limited or its affiliates.
+//
+//          (C) COPYRIGHT 2013-2017 ARM Limited or its affiliates.
+//              ALL RIGHTS RESERVED
+//
+//   This entire notice must be reproduced on all copies of this file
+//   and copies of this file may only be made by a person if such person is
+//   permitted to do so under the terms of a subsisting license agreement
+//   from ARM Limited or its affiliates.
+//----------------------------------------------------------------------------
+
+#ifdef MBED_CLOUD_CLIENT_USER_CONFIG_FILE
+#include MBED_CLOUD_CLIENT_USER_CONFIG_FILE
+#endif
+
+#include <stdint.h>
+
+#ifdef MBED_CLOUD_DEV_UPDATE_ID
+const uint8_t arm_uc_vendor_id[] = {
+    0xa8, 0x73, 0xb7, 0xd2, 0x52, 0xe, 0x5b, 0xbd, 0x86, 0x5e, 0xa6, 0xb0, 0xee, 0xd5, 0xe9, 0xa
+};
+const uint16_t arm_uc_vendor_id_size = sizeof(arm_uc_vendor_id);
+
+const uint8_t arm_uc_class_id[]  = {
+    0xb0, 0xbc, 0xbb, 0x35, 0x63, 0xfc, 0x59, 0xbd, 0xa2, 0xf5, 0xba, 0xe8, 0xa7, 0x75, 0x2, 0xf6
+};
+const uint16_t arm_uc_class_id_size = sizeof(arm_uc_class_id);
+#endif
+
+#ifdef MBED_CLOUD_DEV_UPDATE_CERT
+const uint8_t arm_uc_default_fingerprint[] =  {
+    0xff, 0xdf, 0x89, 0xd2, 0x8f, 0x5e, 0xad, 0xef, 0x0, 0xaf, 0xa2, 0x36, 0x29, 0x18, 0x4b, 0x50,
+    0x83, 0x25, 0x4d, 0x1e, 0xe9, 0x2a, 0x8, 0x9, 0x10, 0x23, 0x84, 0x48, 0x6a, 0x11, 0xbf, 0x6b
+};
+const uint16_t arm_uc_default_fingerprint_size =
+    sizeof(arm_uc_default_fingerprint);
+
+const uint8_t arm_uc_default_subject_key_identifier[] =  {
+    0x18, 0x8b, 0x93, 0x27, 0xe9, 0x6d, 0xc5, 0x9f, 0xfa, 0xbe, 0xad, 0xa6, 0xcd, 0x4f, 0xf8, 0x16,
+    0xdd, 0x19, 0xa3, 0x4d
+};
+const uint16_t arm_uc_default_subject_key_identifier_size =
+    sizeof(arm_uc_default_subject_key_identifier);
+
+const uint8_t arm_uc_default_certificate[] = {
```

Comment:
pour l'intégration dans LekaOS, il faut qu'on ait la liste complète de tous ces certificates, ce qu'ils font, d'où ils viennent, s'ils sont gravés dans le marbre ou si ils peuvent être régénérés, etc.
Et comme dit plus haut, il faut trouver un moyen de pas les commit dans le code.

Reply:
Oui, mais l'ensemble du process de certificats, clés, manifeste n'est pas encore à 100% clair pour moi. Je retirerai au fur et à mesure, le plus rapidement bien sûr.

Reply:
Removed in bee9598

Reply:
ok! on changera quand même toutes les clés et api keys quand on ira sur LekaOS

le commit qui les retire les retire pas vraiment, il faut réécrire l'histoire.

![image](https://user-images.githubusercontent.com/2206544/99001546-55571280-253b-11eb-97c3-931788c931d3.png)